scripts/data_collector/base.py

```diff
@@ -289,7 +289,30 @@ def __init__(
 
     def _executor(self, file_path: Path):
         file_path = Path(file_path)
-        df = pd.read_csv(file_path)
+        df = pd.read_csv(
+            file_path,
+            dtype={self._symbol_field_name: str},
+            keep_default_na=False,
```

Comment:
Can I know what is the intention of adding the parameters `keep_default_na` and `na_values`?
I guessed that it might be to cope with the presence of special data in the raw data, and to test my guess, I did a test and found that it can handle these special characters even without these two parameters.
Here is my test script:
![image](https://user-images.githubusercontent.com/30293408/204711739-92712e53-50de-44cf-8e93-451a767d9ec1.png)

Reply:
The problem was the symbol names 'NA' and 'TRUE'.
![qlib_bug_NA_TRUE](https://user-images.githubusercontent.com/32811724/204730847-94c8f2dc-1aaa-4add-8f3e-03b0f0c5cfa6.PNG)
I could resolve 'TRUE' case by specifying `dtype`, but still 'NA' case didn't work.
So I directly removed 'NA' from pandas's default `na_values`. But I guess this is not a perfect solution. 
Here is a test script for better understanding:
```
import pandas as pd
import io
st = """
NA,1,2,3
TRUE,4,5,6
SYMBOL,N/A,#NA,NULL
"""

# original case
df = pd.read_csv(io.StringIO(st), header=None)
print(df)

# specify dtype: solve 'TRUE'
df2 = pd.read_csv(io.StringIO(st), header=None, dtype={0:str})
print(df2)

# remove 'NA' from default_na_values: solve 'NA'
na_values = pd._libs.parsers.STR_NA_VALUES
na_values.remove('NA')
df3 = pd.read_csv(io.StringIO(st), header=None, keep_default_na=False, na_values=na_values)
print(df3)
```

Reply:
With your reply, I understand the meaning of `keep_default_na` and `na_values` parameters, thanks! But this way of writing makes it difficult to understand the code, I tried a new way of rendering, which makes it easier for the user to understand the code.
Here is my test script:
```
import pandas as pd
import io

st = """
NA,1,2,3
TRUE,4,5,6
SYMBOL,N/A,#NA,NULL
"""

na_cand = [
    "", "#N/A", "#N/A N/A", "#NA", "-1.#IND", "-1.#QNAN", 
    "-NaN", "-nan", "1.#IND", "1.#QNAN", "<NA>", "N/A", 
    "NULL", "NaN", "n/a", "nan", "null",
]

df = pd.read_csv(
    io.StringIO(st),
    header=None,
    dtype={0: str},
    keep_default_na=False,
    na_values={
        0: na_cand,
        1: pd._libs.parsers.STR_NA_VALUES,
        2: pd._libs.parsers.STR_NA_VALUES,
        3: pd._libs.parsers.STR_NA_VALUES,
    }
)
print(df)
```

Reply:
Thank you for better rendering suggestion! I'll refer to it and commit.